### ISSUE TITLE
nodejs_binary warning: node: line 4: grep: command not found

### ISSUE THREAD

**Issue by davido** [bazelbuild/rules_nodejs#1638]
After migrating Gerrit Code Review (Polymer 3) to `rules_nodejs`, we are seeing this warning when running `rollup-bin` rule:

```python
nodejs_binary(
    name = "rollup-bin",
    data = [
        "@tools_npm//:node_modules",
    ],
    # The entry point must be "@tools_npm:node_modules/rollup/dist/bin/rollup",
    # But bazel doesn't run it correctly with the following command line:
    # bazel test --test_env=GERRIT_NOTEDB=ON --spawn_strategy=standalone \
    #    --genrule_strategy=standalone --test_output errors --test_summary detailed \
    #    --flaky_test_attempts 3 --test_verbose_timeout_warnings --build_tests_only \
    #    --subcommands //...
    # This command line appears in Gerrit CI.
    # For details, see comment in rollup-runner.js file
    entry_point = "//tools/node_tools:rollup-runner.js",
)
```

The warning is:

```
SUBCOMMAND: # //polygerrit-ui/app:gr-app-bundle-html [action 'Action polygerrit-ui/app/gr-app-bundle-html.html', configuration: c49a435eb19488377c2d099ba8f1d449e4978af408446d7e2046b35a50133db7]
(cd /home/davido/.cache/bazel/_bazel_davido/5c01f4f713b675540b8b424c5c647f63/execroot/gerrit && \
  exec env - \
  bazel-out/host/bin/tools/node_tools/legacy/crisper-bin.sh -s bazel-out/k8-fastbuild/bin/polygerrit-ui/app/gr-app-bundle-html_tmp.html -h bazel-out/k8-fastbuild/bin/polygerrit-ui/app/gr-app-bundle-html.html -j bazel-out/k8-fastbuild/bin/polygerrit-ui/app/gr-app.js --always-write-script '--script-in-head=false')
INFO: From Bundling JavaScript polygerrit-ui/app/gr-app-bundle-js.js [rollup]:
/home/davido/.cache/bazel/_bazel_davido/5c01f4f713b675540b8b424c5c647f63/sandbox/linux-sandbox/6/execroot/gerrit/bazel-out/host/bin/tools/node_tools/rollup-bin.sh.runfiles/build_bazel_rules_nodejs/internal/node/_node_bin/node: line 4: grep: command not found

bazel-out/k8-fastbuild/bin/polygerrit-ui/app/gr-app-prebundling-srcs/entry.js → bazel-out/k8-fastbuild/bin/polygerrit-ui/app/gr-app-bundle-js.js...
created bazel-out/k8-fastbuild/bin/polygerrit-ui/app/gr-app-bundle-js.js in 12.7s
```

Reproducer: Clone Gerrit@HEAD from https://gerrit.googlesource.com/gerrit, and run:

```
  $ bazel polygerrit-ui/app:gr-app-bundle-html
```

This issue was also reported here: [1]. This happens on Linux here. `rules_nodejs` version currently used is: 1.2.0.

[1] https://bugs.chromium.org/p/gerrit/issues/detail?id=12340

**Comment by alexeagle** [comment]
1.2.4 includes https://github.com/bazelbuild/rules_nodejs/commit/9fb080b
This seems closely related, could you test if that fixes it?

**Comment by davido** [comment]
Thanks, the upgrade to 1.2.4 fixed the problem it indeed.
